rules/S6423/csharp/metadata.json

```
@@ -0,0 +1,18 @@
+{
+  "title": "Azure Functions should log all failures.",
+  "type": "CODE_SMELL",
+  "status": "ready",
+  "remediation": {
+    "func": "Constant\/Issue",
+    "constantCost": "5min"
+  },
+  "tags": [
+    "error-handling"
+  ],
+  "defaultSeverity": "Major",
+  "ruleSpecification": "RSPEC-6423",
+  "sqKey": "S6423",
+  "scope": "All",
```

See also https://github.com/SonarSource/sonar-dotnet/pull/5645#discussion_r870281579
```suggestion
  "scope": "Main",
```